string name
sensor_msgs/Image[] images